server/utils.js

```diff
@@ -78,3 +78,17 @@ utils.decodeSpecialChar = function(url) {
   url = url.replace(/%3F/g, '?');
   return url;
 };
+
+/**
+ * Generate session ID.
+ * @return {string} Session ID.
+ */
+utils.guid = function() {
+  var s4 = function() {
+    return Math.floor((1 + Math.random()) * 0x10000)
+      .toString(16)
+      .substring(1);
+  };
+  return s4() + s4() + '-' + s4() + '-' + s4() + '-' +
```

Comment:
Because hash a random integer just need 
```js
CryptoJS.SHA256(Math.random()).toString();
```
while your solution has smelly code with many s4()...